src/ansys/fluent/core/session_solver.pyi

```diff
@@ -21,4 +21,4 @@ class Solver:
     def read_case(self, file_name: str): ...
     def write_case(self, file_name: str): ...
     @property
-    def settings(self) -> settings_root: ...
+    def settings(self) -> settings_root.root: ...
```

Comment:
Couple of issues in static intellisense in the existing code, probably will merge them separately.